src/python_testing/TC_CADMIN_1_11.py

```diff
@@ -28,7 +28,7 @@
 #       --trace-to perfetto:${TRACE_TEST_PERFETTO}.perfetto
 #       --PICS src/app/tests/suites/certification/ci-pics-values
 #     factory-reset: true
-#     quiet: false
+#     quiet: true
```

Comment:
Just curious, but this seems like an unrelated change?

Reply:
Hello Rene,
You are correct, this change was needed because the above test TC_CADMIN_1_11 appears to have gotten merged with verbose logging set to enabled for CI test runs which it should be set to disabled. 
Please note that I was unfortunately the one that wrote the above test and appears I forgot to swap verbose logging back to disabled for CI args before the test got merged.

Reply:
it is, but the change is correct

Reply:
It just means to not print the output from the test if the test passes. It makes the logs fit in the CI output frame.